examples/typescript/main.ts

```diff
@@ -0,0 +1,15 @@
+import { lift, main, std } from "ts4w";
+import { cloud } from "@winglang/sdk";
+import { equal } from "node:assert";
+
+main((app) => {
+  let bucket = new cloud.Bucket(app, "Bucket");
+
+  bucket.addObject("hello", "Hello World from lifted Bucket!");
+
+  new std.Test(app, "test", lift({bucket}).inflight(async ({bucket}) => {
+    console.log("in test")
+    const hello = await bucket.get("hello");
+    equal(hello, "Hello World from lifted Bucket!");
```

Comment:
The primary limitation of inflights is that you can't use variables from the upper scopes. They either have to be lifted or directly imported from within the inflight. 

Arbitrary functions (like `equal`) currently can't be lifted so it'll have to be imported:

```suggestion
    const { equal } = require("node:assert");
    console.log("in test")
    const hello = await bucket.get("hello");
    equal(hello, "Hello World from lifted Bucket!");
```

The inflights also don't support dynamic import (until we switch off using `vm`), so require is needed.